platform/darwin/src/MGLStyle.mm

```diff
@@ -625,12 +627,26 @@ - (void)localizeLabelsIntoLocale:(nullable NSLocale *)locale {
     }];
 }
 
+- (void)setAccessiblePlaceSourceIdentifiers:(NSSet *)accessiblePlaceSourceLayerIdentifiers {
+    if (![_accessiblePlaceSourceIdentifiers isEqualToSet:accessiblePlaceSourceLayerIdentifiers]) {
+        _accessiblePlaceSourceIdentifiers = accessiblePlaceSourceLayerIdentifiers;
+    }
+}
+
+- (NSSet *)accessiblePlaceSourceIdentifiers {
+    return _accessiblePlaceSourceIdentifiers;
+}
```

Comment:
I’m not seeing anything out of the ordinary about this getter and setter, so I think it should be OK to leave out this custom implementation.